### PR TITLE
CLC-4579 Generate report on bCourses TurnItIn usage as CSV

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.ets.berkeley.edu/jira/browse/CLC-4579

This lets us produce a one-off report from the console. If it meets requirements, a cron job script can land in the next sprint.

### CODE REVIEW

File: app/models/canvas/turnitin_reporter.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    def self.print_current_term
+      current_term_id = Canvas::Proxy.current_sis_term_ids.first
+      worker = TurnitinReporter.new(current_term_id)
+      csv_filename = worker.generate_csv
+      parsed_csv = CSV.read(csv_filename, {headers: true})
+      print parsed_csv.to_csv
+    end
+
+    def initialize(sis_term_id)
+      super()
```
Comment (paulkerschen) [inline app/models/canvas/turnitin_reporter.rb]: @raydavis My only nit to pick is that the empty parentheses don't seem very Ruby.

Comment (raydavis) [inline app/models/canvas/turnitin_reporter.rb]: @paulkerschen , fair cop. (I think I had an argument there in an earlier draft.) I'll amend.

Comment (paulkerschen) [inline app/models/canvas/turnitin_reporter.rb]: @raydavis, ack, I misled you. `super` is magic and, without empty parentheses, will try to pass the same arguments it was called with. My bad!

Comment (raydavis) [inline app/models/canvas/turnitin_reporter.rb]: Ruby and its magic!

**Pull request merged**